### ISSUE TITLE
Wikipedia ads.txt article is blocked

### ISSUE THREAD

**Issue by NickLion** [easylist/easylist#5459]
### List the website(s) you're having issues:

`https://en.wikipedia.org/wiki/Ads.txt`

### What happens?

URL is completely disabled with message:
`Blocked by /ads.txt (EasyList)`

### List Subscriptions you're using:

EasyList

### Your settings

- OS/version: openSUSE Tumbleweed x86_64
- Browser/version: Falkon 3.1.99
- Adblock Extension/version: Built-in Falkon AdBlock plugin

### Other details:

Filter `/ads.txt` should not prevent accessing wikipedia article about this URL.

**Comment by peace2000** [comment]
I don't have any issues. Is your list updated?

Easylist has this whitelist filter that activates for me at least:

`@@||wikipedia.org^$generichide`

**Comment by peace2000** [comment]
That whitelist rule is active for me in Falcon but still that's blocked. Maybe software issue?

![kuva](https://user-images.githubusercontent.com/17256841/83537236-f5c0f980-a4fc-11ea-9de3-97e841775a5c.png)

**Comment by NickLion** [comment]
Yeah, it looks like it's a Falkon problem (I can't find processing for generichide option), not EasyList, sorry.
